scripts/west_commands/runners/bossac.py

```diff
@@ -42,8 +45,54 @@ def do_add_parser(cls, parser):
 
     @classmethod
     def do_create(cls, cfg, args):
+        # BOSSA means there's a bootloader so always fetch the flash address
+        args.dt_flash = True
+        build_conf = BuildConfiguration(cfg.build_dir)
+        flash_address = cls.get_flash_address(args, build_conf, None)
+
         return BossacBinaryRunner(cfg, bossac=args.bossac,
-                                  port=args.bossac_port, offset=args.offset)
+                                  port=args.bossac_port, offset=args.offset,
+                                  flash_address=flash_address)
+
+    def read_help(self):
+        """Run bossac --help and return the output as a list of lines"""
+        self.require(self.bossac)
+        try:
+            self.check_output([self.bossac, '--help'])
```

Comment:
Might be nice to add a comment saying that `bossac --help` exits nonzero. Not asking for a change, though.